config/locales/en.yml

```diff
@@ -892,19 +917,20 @@ en:
       before_start_heading: Before you start
       before_start_paragraphs:
         - It should take about 15 minutes to answer all the questions.
-        - "You'll need information from your client (and any partner) about their:"
+        - "You'll need information about your client's (and their partner, if applicable.):"
```

Comment:
The sentence construction here is a bit tortured, but if we are going to do it like this, `partner` should be `partner's`, and there shouldn't be a full stop in side the parenthesis. I'd recommend going back to Chris and asking exactly how this sentence should read, because I don't think this one has been completely thought out.